athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/data/S3BlockSpiller.java

```diff
@@ -362,6 +362,7 @@ protected SpillLocation write(Block block)
             // only sees the InputStream wrapper.
             ObjectMetadata objMeta = new ObjectMetadata();
             objMeta.setContentLength(bytes.length);
+            objMeta.setSSEAlgorithm(ObjectMetadata.AES_256_SERVER_SIDE_ENCRYPTION);
```

Comment:
This is not necessary, just use this environment variable with this value:

```
spill_put_request_headers = {"x-amz-server-side-encryption" : "AES256"}
```

https://docs.aws.amazon.com/athena/latest/ug/connectors-cloudera-impala.html

Reply:
Thanks that make sense